src/Npgsql/Internal/PgReader.cs

```diff
@@ -560,9 +560,10 @@ internal async ValueTask Consume(bool async, int? count = null, CancellationToke
     public void Consume(int? count = null) => Consume(async: false, count).GetAwaiter().GetResult();
     public ValueTask ConsumeAsync(int? count = null, CancellationToken cancellationToken = default) => Consume(async: true, count, cancellationToken);
 
+    bool StreamActive => _userActiveStream is { IsDisposed: false };
     internal void ThrowIfStreamActive()
     {
-        if (_userActiveStream is { IsDisposed: false})
+        if (StreamActive)
```

Comment:
Seeing some other places where StreamActive could be used instead of the full check

Reply:
Will search and replace.